sys/include/suit.h

```diff
@@ -52,7 +52,9 @@ extern "C" {
 /**
  * @brief   Maximum number of components supported in a SUIT manifest
  */
+#ifndef SUIT_COMPONENT_MAX
 #define SUIT_COMPONENT_MAX                  (1U)
+#endif
```

Comment:
Is this a configuration? If so add `CONFIG_`

Reply:
If it isn't now, it will be in the future. Prefix added!